src/PointwiseFunctions/AnalyticData/GrMhd/BondiHoyleAccretion.hpp

```diff
@@ -0,0 +1,260 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <limits>
+
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "Options/Options.hpp"
+#include "PointwiseFunctions/GeneralRelativity/KerrSchildCoords.hpp"
+#include "PointwiseFunctions/Hydro/EquationsOfState/EquationOfState.hpp"
+#include "PointwiseFunctions/Hydro/EquationsOfState/PolytropicFluid.hpp"  // IWYU pragma: keep
+#include "PointwiseFunctions/Hydro/Tags.hpp"
+#include "Utilities/MakeArray.hpp"  // IWYU pragma: keep
+#include "Utilities/TMPL.hpp"
+#include "Utilities/TaggedTuple.hpp"
+
+// IWYU pragma:  no_include <pup.h>
+
+/// \cond
+namespace PUP {
+class er;  // IWYU pragma: keep
+}  // namespace PUP
+/// \endcond
+
+namespace grmhd {
+namespace AnalyticData {
+
+/*!
+ * \brief Analytic initial data for axially symmetric Bondi-Hoyle accretion.
+ *
+ * In the context of studying Bondi-Hoyle accretion, i.e. non-spherical
+ * accretion on to a moving Kerr black hole, this class implements the method
+ * proposed by Font & Ibáñez (1998) \ref font_98 "[1]" to initialize the GRMHD
+ * variables. The fluid quantities are initialized with their (constant) values
+ * far from the black hole, e.g. \f$\rho = \rho_\infty\f$. Here we assume a
+ * polytropic equation of state, so only the rest mass density, as well as the
+ * polytropic constant and the polytropic exponent, are provided as inputs.
+ * The spatial velocity is initialized using a field that ensures that the
+ * injected gas reproduces a continuous parallel wind at large distances.
+ * The direction of this flow is chosen to be along the black hole spin.
+ * In spherical Kerr-Schild coordinates,
+ *
+ * \f{align*}
+ * v^r &= \frac{1}{\sqrt{\gamma_{rr}}}v_\infty \cos\theta\\
+ * v^\theta &= -\frac{1}{\sqrt{\gamma_{\theta\theta}}}v_\infty \sin\theta\\
+ * v^\phi &= 0.
+ * \f}
+ *
+ * where \f$\gamma_{ij} = g_{ij}\f$ is the spatial metric, and \f$v_\infty\f$
+ * is the flow speed far from the black hole. Finally, following the work by
+ * Penner (2011) \ref penner_11 "[2]",
+ * the magnetic field is initialized using Wald's solution to Maxwell's
+ * equations in Kerr black hole spacetime. In spherical Kerr-Schild coordinates,
```

Comment:
You were avoiding the term "spherical Kerr-Schild coordinates" in a previous commit because it was ambiguous.  Do you want to use it here?